README.md

```diff
@@ -1,5 +1,7 @@
 Form in modal implemented for book#edit in books/index.html.erb
 
+<This line is added by Anshuman>
+
```

Comment:
hey, 
cool, nice to meet you, but unfortunately have to work more on this repo, can't merge this